docs/src/contrib/debug.md

```diff
@@ -12,6 +12,8 @@ There are different flavours of manifests shipped to help with local development
 
 ### Development build
 
+You must first build the binary with `make build`, the following steps will then add it to the Docker image.
+
 The main difference it that image defined in manifest is `metacontrollerio/metacontroller:dev`, therefore:
 * apply dev manifests - `kubectl apply -k manifests/dev`
 * build docker image with command - `docker build -t metacontrollerio/metacontroller:dev -f Dockerfile .`
```

Comment:
actually currently it is done by `make image` step - https://github.com/metacontroller/metacontroller/blob/master/Makefile#L44